packages/web3-eth2-beaconchain/test/schemaMethods.test.ts

```diff
@@ -0,0 +1,270 @@
+import { ETH2BeaconChain } from '../src/index'
+
+// Jest doesn't have a native method to test if value is boolean
+expect.extend({
+    toBeBoolean(received) {
+        return typeof received === 'boolean' ? {
+            message: () => `expected ${received} to be boolean`,
+            pass: true
+        } : {
+            message: () => `expected ${received} to be boolean`,
+            pass: false
+        }
+    }
+})
+
+let eth2BeaconChain
+
+beforeAll(() => {
+    const provider = 'http://127.0.0.1:9596' // default port for Lodestar
+    eth2BeaconChain = new ETH2BeaconChain(provider)
+})
+
+it('getGenesis', async () => {
+    const expectedResponse = {
+        genesis_validators_root: expect.stringMatching(/0x[a-f|A-F|\d]{64}/),
+        genesis_time: expect.stringMatching(/\d{10}/),
+        genesis_fork_version: expect.stringMatching(/0x[a-f|A-F|\d]{8}/)
+    }
+
+    const response = await eth2BeaconChain.getGenesis()
+    expect(response).toMatchObject(expectedResponse)
+})
+
+it('getHashRoot - NOT IMPLEMENTED IN LODESTAR', async () => {
```

Comment:
Since these are Lodestar specific, perhaps renaming the file might be helpful

Reply:
These routes aren't Lodestar specific, I've just been using Lodestar because it implements the official spec. I've take out the `...IN LODESTAR` from the test comment and just left it as `...NOT IMPLEMENTED`